src/pages/LinkedInLearning/LinkedInLearningPage.jsx

```diff
@@ -0,0 +1,37 @@
+import Button from "@mui/material/Button";
+import { Box } from "@mui/system";
+import PageLayout from "../../components/PageLayout";
+
+/**
+ * Renders the LinkedIn Learning page.
+ * @returns {JSX.Element} LinkedIn Learning page component
+ */
+function LinkedInLearning() {
+  return (
+    <PageLayout viewportPage title="LinkedIn Learning">
+      <Box
+        sx={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "calc(100% - 60px)",
+        }}
+      >
+        <video width="80%" height="90%" controls>
+          <source src="" type="video/mp4" />
```

Comment:
Why didn't you implemented directly a way to pass the video source as a prop into the component?

Reply:
Changed